presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix memory over-accounting for stages where aggregation is followed by other non-trivial work.
+* Fix formatting in EXPLAIN ANALYZE output.
+* Fix planning error for `LEFT JOIN` converted to `INNER JOIN` without equi-conditions.
+* Improve error message when query contains unsupported correlated subquery.
+* Improve performance of ``CAST(json_parse(...) AS ...)``
+* Add :func:`map_from_entries` and :func:`map_entries` function
+* Make possible for `CHAR` functions (like `subst`) to  have empty spaces at the end of
+  the result.
+* Spill to disk for aggregation uses revocable memory. Now spilling takes place only if
```

Comment:
The first sentence is an implementation detail that can be removed as it doesn't add to the user's understanding of the change. We can rewrite this as
```
* Improve spilling to only occur when the cluster runs out of memory.